README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+    - [DEMO LINK](https://<KrisMakarovska>.github.io/js_task_parse_list_DOM/)
```

Comment:
```suggestion
    - [DEMO LINK](https://KrisMakarovska.github.io/js_task_parse_list_DOM/)
```
Please fix a link in your PR description, so it opens correctly 
![image](https://user-images.githubusercontent.com/96409563/209652892-063dcc09-3761-43c0-a081-7c1d8723aaf1.png)
Btw, even with the right format it still doesn't work. Make sure that you've deployed your page
![image](https://user-images.githubusercontent.com/96409563/209653005-7a7dbbac-d61e-44a5-8592-9a049cb85d2b.png)

Reply:
fixed